api/src/test/java/org/openmrs/DurationTest.java

```diff
@@ -98,7 +99,8 @@ public void addToDate_shouldFailWhenUnitIsRecurringAndFrequencyIsUnknown() throw
 		OrderFrequency frequency = null;
 		
 		expectedException.expect(APIException.class);
-		expectedException.expectMessage(Matchers.is("Duration.error.frequency.null"));
+		expectedException.expectMessage(Matchers.is(Context.getMessageSourceService().getMessage(
```

Comment:
I posted another comment saying that you ignore that comment but i don't know what happened to it, ignore it